app/services/summary_service.rb

```diff
@@ -39,17 +45,19 @@ def self.summary(summary_type, prison, page, params)
           bucket.items << offender if summary_type == :unallocated
           counts[:unallocated] += 1
         end
-      else
+      elsif offender.awaiting_allocation_for > 2
```

Comment:
Would be nice if this were configurable or at least a named constant. Are we sure that awaiting_allocation_for is a number of days. If so it needs a better name